Mathlib/LinearAlgebra/BilinearForm/Properties.lean

```diff
@@ -55,8 +54,7 @@ namespace BilinForm
 
 
 /-- The proposition that a bilinear form is reflexive -/
-def IsRefl (B : BilinForm R M) : Prop :=
-  ∀ x y : M, B x y = 0 → B y x = 0
+def IsRefl (B : BilinForm R M) : Prop := LinearMap.IsRefl (toLin B)
```

Comment:
Can the `toLin`s be dropped now that the other PR landed?

Reply:
Yes. In fact `toLin` is deprecated in #12078.